contents.json

```diff
@@ -5766,6 +5766,13 @@
       "description": "Use any custom view as custom callout view for MKMapView with cool animations. Use any image as annotation view.",
       "homepage": "https://github.com/okhanokbay/MapViewPlus",
       "tags": ["map", "customcalloutview", "calloutview", "annotationview", "swift", "MapKit", "MKMapView"]
+    },
+    {
+      "title": "TraceLog",
+      "category": "logging",
+      "description": "Dead Simple: logging the way it's meant to be!  Runs on ios, osx, and linux.",
```

Comment:
Can you update to say iOS, macOS and Linux ?

Reply:
Done.